printer.go

```diff
@@ -58,6 +92,36 @@ func Default() (string, error) {
 	return syscall.UTF16ToString(b), nil
 }
 
+func GetDefaultPrinterType() (string, error) {
+	printerName, _ := Default()
+	return GetPrinterType(printerName)
+}
+
+func GetPrinterType(printerName string) (string, error) {
```

Comment:
Instead of creating package function, I suggest, you add new method to Printer struct. This way new method will not need printerName parameter. You will also not need GetFeaultPrinterType function, because you could use printer.Default.... instead.

Reply:
That is a nice a idea !